src/languages/java.md

```diff
@@ -0,0 +1,3 @@
+# Java
+
+Java support is in Alpha and not generally available. Contact sales [https://platform.sh/contact](https://platform.sh/contact) if you are interested in running Java based applications on Platform.sh
```

Comment:
Needs a period on the end. :smile: I'm also still a bit wary of listing Java yet, since it's so finicky.